### DISCUSSION TITLE
dataPlane报insufficient credentials code:401错误，请教怎么解决呀

### DISCUSSION THREAD

**Discussion by gzlboy** [apache/apisix#9738] [Q&A]
etcd版本：3.5.0
apisix版本：3.3.0

使用bitnam的apisix chart安装的

controlPlane正常
etcd里执行etcdctl get --prefix /apisix --user root:xxxx也正常返回

dataPlane的配置：
![image](https://github.com/apache/apisix/assets/1492060/43ecaa40-69d9-4be9-9bc6-324b992a6e58)

dataPlane的log：
2023/06/28 07:00:46 [error] 11#11: *78323 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/plugin_configs, context: ngx.timer
2023/06/28 07:00:47 [error] 8#8: *77692 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/routes, context: ngx.timer
2023/06/28 07:00:48 [error] 7#7: *71060 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/global_rules, context: ngx.timer
2023/06/28 07:00:50 [error] 11#11: *70975 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/routes, context: ngx.timer
2023/06/28 07:00:50 [error] 8#8: *72207 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/services, context: ngx.timer
2023/06/28 07:00:51 [error] 7#7: *73339 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/plugin_metadata, context: ngx.timer
2023/06/28 07:00:52 [error] 11#11: *73330 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/plugin_metadata, context: ngx.timer
2023/06/28 07:00:52 [error] 11#11: *73163 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/global_rules, context: ngx.timer
2023/06/28 07:00:52 [warn] 8#8: *78858 [lua] v3.lua:245: _request_uri(): https://apisix-release-control-plane:9280: failed to parse domain: failed to parse domain. Retrying, context: ngx.timer
2023/06/28 07:00:52 [error] 8#8: *78858 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/plugin_metadata, context: ngx.timer
2023/06/28 07:00:53 [error] 11#11: *78132 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/consumers, context: ngx.timer
2023/06/28 07:00:55 [error] 11#11: *74180 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/plugins, context: ngx.timer
2023/06/28 07:00:56 [error] 8#8: *74766 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/protos, context: ngx.timer
2023/06/28 07:00:57 [error] 11#11: *73354 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/ssls, context: ngx.timer
2023/06/28 07:00:57 [error] 11#11: *74474 [lua] config_etcd.lua:634: failed to fetch data from etcd: insufficient credentials code: 401,  etcd key: /apisix/secrets, context: ngx.timer

请教可能是什么问题呀

**Comment by 0x101011** [comment]
你apisix部署环境和etcd部署环境网络之间有问题吗？ apisix本地配置一下https://apisix-release-control-plane:9280 这个host？
